lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -178,8 +178,18 @@ class CategoryRenderer {
     gaugeContainerEl.appendChild(gaugeEl);
 
     if (category.description) {
+      const tooltipTmpl = this.dom.cloneTemplate('#tmpl-lh-category-header-description-tooltip',
+        this.templateContext);
+      const tooltipEl = this.dom.find('.tooltip-boundary', tooltipTmpl);
       const descEl = this.dom.convertMarkdownLinkSnippets(category.description);
-      this.dom.find('.lh-category-header__description', tmpl).appendChild(descEl);
+      this.dom.find('.tooltip', tooltipEl).appendChild(descEl);
+
+      // Wrap the label so the tooltip icon can be positioned relative to it.
+      const gaugeWrapper = this.dom.find('.lh-gauge__wrapper', gaugeContainerEl);
+      const labelWrapper = this.dom.createChildOf(gaugeWrapper, 'div', 'lh-gauge__label-wrapper');
+      const gaugeLabelEl = this.dom.find('.lh-gauge__label', gaugeContainerEl);
+      labelWrapper.appendChild(gaugeLabelEl);
+      labelWrapper.appendChild(tooltipEl);
```

Comment:
for some reason I can't for the life of me follow what this is doing :) Taking stuff in the template, appending it to the wrapper, than putting the wrapper into what was in the template? Can the `labelWrapper` just live in the template? Seems like that it might simplify this dance

Reply:
The issue is that the tooltip must be placed relative to the label, and the label is in `tmpl-lh-gauge`, which is shared by the category header gauges and the main header gauge.

Perhaps `renderScoreGauge` could also take a description, the tooltip stuff moved to the `tmpl-lh-gauge` template, and hide the tooltip if the description is empty.